tests/integration/test_pip_packages.py

```diff
@@ -31,14 +31,15 @@ def test_all_pip_packages(env_name, test_env, tmpdir):
         payload={"repo": env_data["repo"], "ref": env_data["ref"], "pkg_managers": ["pip"]}
     )
     completed_response = client.wait_for_complete_request(initial_response)
+    response_data = utils.sort_cachito_response(completed_response.data)
     assert completed_response.status == 200
-    assert completed_response.data["state"] == "complete"
-    assert completed_response.data["state_reason"] == "Completed successfully"
+    assert response_data["state"] == "complete"
+    assert response_data["state_reason"] == "Completed successfully"
 
     expected_package_params = [env_data["package"]]
```

Comment:
You should sort expected data so it does not depend on these packages/dependencies are sorted in test_env_vars.yaml file.